src/Raven.Server/Json/BlittableJsonTextWriterExtensions.cs

```diff
@@ -566,7 +568,9 @@ public static async Task<(long NumberOfResults, long TotalDocumentsSizeInBytes)>
 
                 writer.WriteEndObject();
             }
-            var revisionByCv = result.GetRevisionIncludesByChangeVector();
+
+            /*TODO:
```

Comment:
?

Reply:
DevelopmentHelper.ToDO

Reply:
There was no abstract method for `QueryResultServerSide`. Added.